docs/docs-graphql/schema.graphqls

```diff
@@ -1,3 +1,6 @@
+"""Indicates that the argument defaults to a contextual path."""
+directive @defaultPath(path: String!, ignore: [String!]!) on ARGUMENT_DEFINITION
```

Comment:
This is exposed and annotated on contextual args so that the inner MCP implementation can determine which arguments are contextual and satisfy them using the Env hostfs.